scripts/global_ens/stats/exevs_global_ens_headline_grid2grid_stats.sh

```diff
@@ -37,7 +37,33 @@ verify_type=$3
 #############################################################
 
 if  [ $ens = gefs ] || [ $ens = naefs ] || [ $ens = gfs ] ; then
-  if [ $verify_type = upper ] ; then 
+  if [ $verify_type = upper ] ; then
+
+   if [ $ens = gfs ] || [ [ $ens = gefs ] ; then
+    if [ ! -s ${COMIN}.${VDATE}/gefs/gfsanl.t00z.grid3.f000.grib2 ] ; then
+         	       
+       export subject="GFS analysis data missing for $ens headline stat job"
+       export maillist=${maillist:-'geoffrey.manikin@noaa.gov,binbin.zhou@noaa.gov'}
+       echo "Warning: No GFS analysis available for ${VDATE}" > mailmsg 
+       echo Missing file is ${COMIN}.${VDATE}/gefs/gfsanl.t00z.grid3.f000.grib2  >> mailmsg
+       echo "Job ID: $jobid" >> mailmsg
+       cat mailmsg | mail -s "$subject" $maillist
+       exit
+    fi
+   fi
+
+   if [ $ens = naefs ] ; then 
+      if [ ! -s ${COMIN}.${VDATE}/cmce/cmcanl.t00z.grid3.f000.grib2 ] || [ ! -s ${COMIN}.${VDATE}/gefs/gfsanl.t00z.grid3.f000.grib2 ] ; then
+        export subject="GFS or CMC analysis data missing for $ens headline stat job"
+        export maillist=${maillist:-'geoffrey.manikin@noaa.gov,binbin.zhou@noaa.gov'}
```

Comment:
@BinbinZhou-NOAA this line (58) needs to be removed now that the variable is exported from the ecf script.